framework/test/unit/modules/interoperability/internal_method.spec.ts

```diff
@@ -710,34 +737,131 @@ describe('Base interoperability internal method', () => {
 			const ccu = {
 				...ccuParams,
 				certificate: codec.encode(certificateSchema, certificate),
-				activeValidatorsUpdate: [
-					{ blsKey: Buffer.from([0, 0, 0, 1]), bftWeight: BigInt(5) },
-					{ blsKey: Buffer.from([0, 0, 0, 1]), bftWeight: BigInt(5) },
-					{ blsKey: Buffer.from([0, 0, 3, 0]), bftWeight: BigInt(5) },
-				],
+				activeValidatorsUpdate: {
+					blsKeysUpdate: [
+						Buffer.from([0, 0, 0, 0]),
+						Buffer.from([0, 0, 0, 1]),
+						Buffer.from([0, 0, 2, 0]),
+					],
+					bftWeightsUpdate: [BigInt(1), BigInt(3), BigInt(4)],
+					bftWeightsUpdateBitmap: Buffer.from([14]),
+				},
 			};
+			await chainValidatorsSubstore.set(methodContext, ccu.sendingChainID, {
+				activeValidators: [{ blsKey: Buffer.from([0, 0, 2, 0]), bftWeight: BigInt(2) }],
+				certificateThreshold: BigInt(1),
+			});
 
 			await expect(
 				mainchainInteroperabilityInternalMethod.verifyValidatorsUpdate(methodContext, ccu),
 			).rejects.toThrow('Keys have duplicated entry');
 		});
 
+		it('shoud reject if bftWeightsUpdateBitmap does not corresponds to the bftWeightsUpdateBitmap', async () => {
```

Comment:
I am a bit confused by this description, maybe we want to say 'shoud reject if bftWeightsUpdateBitmap does not corresponds to the bftWeightsUpdate'?